packages/workspace/src/generators/ci-workflow/__snapshots__/ci-workflow.spec.ts.snap

```diff
@@ -1006,22 +1024,20 @@ jobs:
           filter: tree:0
           fetch-depth: 0
 
-      - uses: pnpm/action-setup@v4
+      # Cache node_modules
```

Comment:
The order should not have changed here and we should continue using the pnpm setup action since that's what they still recommend on the docs.